src/modules/universal-discovery/components/content-meta-preview/css/content.meta.preview.component.css

```diff
@@ -116,3 +117,70 @@
 .c-meta-preview__translation {
     display: block;
 }
+
+.c-meta-preview__top-wrapper {
+    width: 190px;
+    margin: auto;
+    display: flex;
+}
+
+.c-meta-preview__content-bookmark {
+    flex: 0 0 32px;
+    height: 30px;
+}
+
+.c-meta-preview__content-bookmark .ez-add-to-bookmarks__icon-wrapper {
+    cursor: pointer;
+}
+
+.c-meta-preview__content-bookmark .ez-add-to-bookmarks__icon-wrapper--add::after,
+.c-meta-preview__content-bookmark .ez-add-to-bookmarks__icon-wrapper--remove::after {
+    content: "";
+    display: block;
+    width: 32px;
+    height: 32px;
+    margin-top: -33px;
+    margin-left: -4px;
+    border-radius: 50%; 
+
+    animation-direction: normal;
+    animation-iteration-count: 1;
+    animation-play-state: running;
+}
+
+.c-meta-preview__content-bookmark .ez-add-to-bookmarks__icon-wrapper--add::after {
+    background-color: transparent;
+    animation: background-to-transparent .25s ease-in;
+}
+
+.c-meta-preview__content-bookmark .ez-add-to-bookmarks__icon-wrapper--remove::after {
+    background-color: #878787;
+    animation: background-to-gray .25s ease-in;
+}
+
+.c-meta-preview__content-bookmark .ez-add-to-bookmarks__icon-wrapper--remove [class^="ez-icon"] {
+    fill: white;
+}
+
+.c-meta-preview__content-bookmark .ez-add-to-bookmarks__icon-wrapper--add [class^="ez-icon"] {
+    fill: #878787;
+}
+
+@keyframes background-to-gray {
```

Comment:
As was discussed on the demo, this should be `background-to-white`? And the star color should be blueish. Is this applied?